### ISSUE TITLE
Using PartialDataLoading has wrong paths

### ISSUE THREAD

**Issue by tinect** [FriendsOfShopware/FroshPlatformThumbnailProcessor#145]
### PHP Version

-

### Shopware Version

6.6.6.1

### Plugin Version

5.2.0

### Actual behaviour

http://localhost/thumbnail/....

### Expected behaviour

http://localhost/media/....

### Steps to Reproduce?

```
$criteria = new Criteria();

$criteria->addFields(['cover.media']);
$criteria->addFilter(new EqualsFilter('productNumber', $productNo));
$criteria->addAssociation('cover.media.thumbnails');

return $this->productRepository->search($criteria, $context)->getEntities()->first()?->get('cover')?->get('media');
```

**Comment by tinect** [comment]
name has been changed https://github.com/shopware/shopware/commit/6e0787d4313d4200a0e494101511ea9d4d24f454

**Comment by tinect** [comment]
Fixed with 5.2.1
